### PR TITLE
Added financial contributors to the README

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or another core contributor signed this repository up for Open Collective. This pull request adds financial contributors from your Open Collective https://opencollective.com/ethical-source ❤️

  What it does:
  - adds a badge to show the latest number of financial contributors
  - adds a banner displaying contributors to the project on GitHub
  - adds a banner displaying all individuals contributing financially on Open Collective
  - adds a section displaying all organizations contributing financially on Open Collective, with their logo and a link to their website

P.S: As with any pull request, feel free to comment or suggest changes.

  Thank you for your great contribution to the Open Source community. You are awesome! 🙌
  And welcome to the Open Collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other Open Source sustainers!

### CODE REVIEW

**Review by zspencer — APPROVED** [review]
Thank you for doing this!

File: README.md [hunk 1]
```diff
@@ -53,3 +55,33 @@ If you are interested in doing a translation, please follow these steps:
...
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/EthicalSource/ethicalsource.dev/graphs/contributors"><img src="https://opencollective.com/ethical-source/contributors.svg?width=890&button=false" /></a>
+
+### Financial Contributors
+
+Become a financial contributor and help us sustain our community. [[Contribute](https://opencollective.com/ethical-source/contribute)]
```
Comment (nateberkopec) [inline README.md:68]: Once this gets merged, we should specify what donations are used for. Not a blocker tho.

**Review by CoralineAda — APPROVED** [review]

**Comment by CoralineAda** [comment]
@monkeywithacupcake one of the badges in the README is a broken image. The url is https://opencollective.com/ethical-source/contributors.svg?width=890&button=false

**Comment by monkeywithacupcake** [comment]
Sometimes the image does not show up. If it continues to break, please reach out on slack or put in an issue on Open Collective front end.

**Pull request merged**